src/components/CreateCard/UpdateCard.js

```diff
@@ -0,0 +1,120 @@
+import React, {useState} from "react";
+import {Form, Icon, Input, Button, Spin, Alert} from "antd";
+import "../CreateCard/AddCard.module.css";
+import {axiosWithAuth} from "../../utils/axios";
+import {updateCard} from "../../state/actions/CardAction";
+import {connect} from "react-redux";
+import styles from "./AddCard.module.css";
+
+function UpdateCard(props) {
+  const [formValues, setFormValues] = useState({
+    deckId: "",
+    questionText: "",
+    answerText: "",
```

Comment:
I'd say refactor this and the AddCard.js into a single component, and make it take initial props. You are repeating yourself with the exception of a couple of strings. 
Also I'm not seeing you populating the form with values you get from the card you need to update. Which is a bad UX